site/topics/memory/memory.rst

```diff
@@ -2,16 +2,94 @@
 Memory
 ******
 
-* Storing data for later use is an important function of a typical digital computer
+* Until now, all circuits discussed made no use of stored information
+* However, storing data for later use is an important function of a typical digital computer
 * Memory can be used to store
 
     * Data, such as the value of a variable
     * Instructions, such as a program
 
 
 
-Bits and Bytes
-==============
+Bits, Bytes, and Data
+=====================
+
+* Before getting into the details on memory, it's important to be clear on encoded and decoded information
+* All values within a typical computer are encoded as binary patterns
+
+* For example, consider the binary sequence ``01100001``
+* This binary pattern is an encoding that can have different meaning depending on how and what it's encoding
+
+    * The pattern could mean :math:`97_{10}` if it's an encoding for an integer
+    * Or it could be an encoding for the letter ``a``
+    * It may encode the colour blue
+    * It could even be a pattern to control circuitry within a CPU to perform addition
+
+
+.. note::
+
+    The word "data" is used to refer to raw values/symbols that have no intrinsic meaning, while the word "information"
+    is used to refer to data that has some meaning within some context.
+
+    For example, the number :math:`3` alone is data, but the number :math:`3` can be information if it's referring to,
+    for instance, the quantity of ducks swimming in a pond.
```

Comment:
I'd be careful here -- this is also data in the right context. In the context of computer memory, encodings are informational, but at higher levels it do be being data.